src/components/votingcenter/CenterContainer.js

```diff
@@ -0,0 +1,19 @@
+import React, { useEffect, useContext } from 'react';
+import {AppContext} from '../app/App';
+
+
+function CenterContainer() {
+  const {state} = useContext(AppContext);
+
+  // this useEffect is just to check the value, you can totally remove it
+  useEffect(() => {
+    console.log(state.address);
+  }, [state.address])
```

Comment:
feel free to remove this, this is just to demonstrate that the address is coming through :)